web/components/answers/answers-panel.tsx

```diff
@@ -193,40 +206,37 @@ function OpenAnswer(props: {
         />
       </Modal>
 
-      <div
-        className="pointer-events-none absolute -mx-2 h-full rounded-tl-md bg-green-600 bg-opacity-10"
-        style={{ width: `${100 * Math.max(prob, 0.01)}%` }}
-      />
-
-      <Row className="my-4 gap-3">
-        <Avatar className="mx-1" username={username} avatarUrl={avatarUrl} />
-        <Col className="min-w-0 flex-1 lg:gap-1">
-          <div className="text-sm text-gray-500">
-            <UserLink username={username} name={name} /> answered
-          </div>
-
-          <Col className="align-items justify-between gap-4 sm:flex-row">
-            <Linkify className="whitespace-pre-line text-lg" text={text} />
-            <Row className="align-items items-center justify-end gap-4">
-              <span
-                className={clsx(
-                  'text-2xl',
-                  tradingAllowed(contract) ? 'text-primary' : 'text-gray-500'
-                )}
-              >
-                {probPercent}
-              </span>
-              <BuyButton
-                className={clsx(
-                  'btn-sm flex-initial !px-6 sm:flex',
-                  tradingAllowed(contract) ? '' : '!hidden'
-                )}
-                onClick={() => setOpen(true)}
+      <Col
+        className={clsx(
+          'bg-greyscale-1 relative w-full rounded-lg transition-all',
+          tradingAllowed(contract)
+            ? 'text-greyscale-7 drop-shadow-sm hover:drop-shadow-md'
+            : 'text-greyscale-5 pointer-events-none'
+        )}
+        onClick={() => setOpen(true)}
+      >
+        <Row className="z-20 -mb-1 justify-between py-2 px-3">
```

Comment:
It's not clear that you can click on these to purchase them (on desktop, at least -- and probably on mobile too.)

I'd at least add a `cursor-pointer`, and it would be really cool to have the word "Buy" or "Predict" or something float in from the left side when you're hovering over the bar!

Not sure how to make this more explicit on mobile...

Reply:
Hmm yeah I thought maybe having a small dropshadow made them look more clickable, I'll continue thinking about it

Reply:
I agree with Austin, I think this looks good but nobody will have any clue how to bet. Why not put a button saying "buy" to the right of the answer?

Reply:
added a buy button!